test-configs.yaml

```diff
@@ -751,35 +735,57 @@ device_types:
       - whitelist: {defconfig: ['defconfig']}
       - blacklist: {kernel: ['v3.']}
 
-  qemu_arm:
-    type: qemu
+  qemu-arm-virt-gicv2:
```

Comment:
I think it's a bit easier to read `qemu_arm-virt-gicv2` to make it clear that it's a variant of qemu, but that's quite subjective.